nextflow/modules/blast.nf

```diff
@@ -24,6 +25,7 @@ process blast {
     """
     else if (task.attempt.toString() == '2')
     """
+      sed -i "s/ /|/" ${fasta}
```

Comment:
Sorry, I don't remember. Why do we need this sed instruction here?

Reply:
I think only to have the complete FASTA ID information in the blast result. Because otherwise, blast would split the ID at the first occurring whitespace.

Reply:
Yes! We need it to ensure consistency across checkV and proteins prediction, thus resolving the issue when multiple prophages were present in the same contig.

Reply:
I see. Then, it would be more convenient to use a different separator for our annotations.
Instead of:
```
>seq1 prophage-475020:518621
```
we should use:
```
>seq1_prophage-475020:518621
```
then we don't have to use sed and all the names will match everywhere